### ISSUE TITLE
Video capture orientation not handled proper

### ISSUE THREAD

**Issue by Aks-4125** [amosyuen/FFmpegVideoRecorder#14]
I have been doing R & D on capturing video in square format and output for the same. Library does meet the requirements but the issue is whenever i capture video with portrait/landscape mode then resulting video is not rotating as per screen.

![device-2018-01-03-162051](https://user-images.githubusercontent.com/29367708/34517715-3215da48-f0a2-11e7-9d57-240a12efb791.png)

Even sometime output comes in gray-scale. 

Please let me know what needs to be done here to overcome orientation problem. If I get success then I will do PR for contribution.

Device info : Moto G play Android 6.0

**Comment by amosyuen** [comment]
@Aks-4125 The rotation should work if the phone rotation lock is off. In that case, before you start recording, when you flip the phone the blue header bar should move to the top of the landscape view and it will be recorded correctly in landscape resolution. If rotation lock is on, it will always record in the Activity orientation. Could you confirm whether the rotation lock was on or off when you recorded?

**Comment by Aks-4125** [comment]
Yes, I have to enable rotation of device to support landscape recording. But, [this library](https://github.com/googlesamples/android-Camera2Video) allows landscape recording when device rotation is lock(only portrait), and resulting landscape recording is properly handled for portrait view.
